docs/api-reference/global-functions/String.md

```diff
@@ -1,19 +1,30 @@
 ---
-description: String String(Byte[] byteArray)
+uid: crmscript_global_string
+title: String
+description: CRMScript String constructor
 intellisense: Void.String
 langref: 1
-keywords: String(Byte[])
+keywords: String(String), String(Byte[]), String(NSStream), String(Byte[],String), String(NSStream,String)
 so.topic: reference
 ---
 
```

Comment:
Considering linking to, or include examples from [data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/string-type.md).